src/courses/forms.py

```diff
@@ -117,16 +117,31 @@ class Meta:
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
+        self.fields['name'].help_text = 'If empty, will instead use First day as it\'s name'
```

Comment:
This should be in the model field's help_text, then the form will use it by default.  Is there a specific reason to put it hear instead?

Reply:
changed